packages/core/src/services/commits/merge.ts

```diff
@@ -35,6 +36,16 @@ export async function mergeCommit(
       )
     }
 
+    const recomputedResults = await recomputeChanges({ commitId }, tx)
+    if (recomputedResults.error) return recomputedResults
+    if (Object.keys(recomputedResults.value.errors).length > 0) {
+      return Result.error(
+        new LatitudeError(
+          'There are errors in the updated documents in this commit',
```

Comment:
What about returning the errors?

Reply:
Why though? It's not being used for anything

Reply:
What about explaining the errors to the user in the UI?

Reply:
That should be shown in the UI **before** trying to actually perform the merge, shouldn't it? 
This should just be the last validation before doing the actual merge action.